spec/classes/mod/dev_spec.rb

```diff
@@ -1,34 +0,0 @@
-require 'spec_helper'
-
-describe 'apache::mod::dev', type: :class do
-  let(:pre_condition) do
-    [
-      'include apache',
-    ]
-  end
-
-  it_behaves_like 'a mod class, without including apache'
-
-  [
-    ['RedHat', '6', 'Santiago', 'Linux'],
-    ['Debian', '8', 'jessie', 'Linux'],
```

Comment:
If we still support Debian 8, we may want to keep this?

Reply:
RedHat 6 as well